bubblegum/program/src/lib.rs

```diff
@@ -500,15 +505,14 @@ fn process_mint_v1<'info>(
         creator_hash.to_bytes(),
     );
     let new_nft = NewNFTEvent {
+        account_type: AccountType::NewNFTEvent,
```

Comment:
In the program parser PR you also mentioned: "We dont need any event except leaf schema".

I'm wondering should I just get rid of all the other event types then, and get rid of the AccountType 1-byte discriminator?

Then on the program parser we can just always assume its a `LeafSchemaEvent`.  This would be much simpler.

The choice is between:
A) Simpler
B) Potential future proofing if we need more info than what LeafSchema can provide.

Reply:
Had brief offline conversation with @austbot while he's travelling.  From my impression, I will leave the 1-byte discriminator account type, but remove the currently unneeded events.  This will future proof us in a couple different ways but also remove currently extraneous code.

Reply:
Updated in latest code to have just `LeafSchemaEvent`